### PR TITLE
Alkwa/interop conceptual docs

### PR DESCRIPTION
**Pull request by alkwa-msft**
Conceptual Docs/Interop Meeting related stories

### CODE REVIEW

**Comment by dmceachernmsft** [comment]
Overall I think looks good but is it possible to link these previews to their docs? image

File: packages/storybook8/stories/Concepts/TeamsInterop/ComplianceBanner/ComplianceBanner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export const ComplianceBanner = ComplianceBannerStory.bind({});
+
+export default {
+  title: 'Concepts/Teams Interop/Compliance Banner',
+  component: ComplianceBanner,
+  argTypes: {
+    callRecordState: { control: 'boolean', defaultValue: false, name: 'Is recording on?' },
+    callTranscribeState: { control: 'boolean', defaultValue: false, name: 'Is transcription on?' }
+  }
+} as Meta;
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Concepts/TeamsInterop/ComplianceBanner/ComplianceBanner.stories.tsx:27]: need to set default values using `args: {}` 
Screenshot 2024-07-29 at 3 42 18 PM

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+export const JoinMeetingWithLink = JoinWithMeetingLinkStory.bind({});
+
+export default {
+  title: 'Concepts/Teams Interop/Join Meeting With Link',
+  component: JoinMeetingWithLink,
+  argTypes: {
+    ...storyControls,
+    ...defaultCallWithChatCompositeHiddenControls
+  }
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx:67]: need to set default values using `args: {}`
Screenshot 2024-07-29 at 3 43 30 PM

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithMeetingId/JoinMeetingWithMeetingId.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+export const JoinMeetingWithMeetingId = JoinWithMeetingIdStory.bind({});
+
+export default {
+  title: 'Concepts/Teams Interop/Join Meeting With Meeting Id',
+  component: JoinMeetingWithMeetingId,
+  argTypes: {
+    ...storyControls,
+    ...defaultCallWithChatCompositeHiddenControls
+  }
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithMeetingId/JoinMeetingWithMeetingId.stories.tsx:68]: need to set default values using `args: {}`

File: packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+export default {
+  title: 'Concepts/Teams Interop/Lobby',
+  component: Lobby,
+  argTypes: {
+    callStateText: controlsToAdd.callStateText,
+    callStateSubText: controlsToAdd.callStateSubText,
+    isVideoReady: controlsToAdd.isVideoReady
+  }
+} as Meta;
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx:24]: Preview not rendering properly:
Screenshot 2024-07-29 at 3 44 54 PM
this is how it looks in storybook6:
Screenshot 2024-07-29 at 3 44 45 PM

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx
Comment (prabhjot-msft) [inline packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx:1]: can we add `index.stories.tsx` to export the story and move the story to separate file `JoinMeetingWithLink.story.tsx`. Same for all the `stories.tsx` files for consistency

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx (hunk 2 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx:67]: will look into this

File: packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx (hunk 4 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx:24]: will look into this

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx:1]: thanks for all of the comments above. checkin' them out now

File: packages/storybook8/stories/Concepts/TeamsInterop/ComplianceBanner/ComplianceBanner.stories.tsx (hunk 1 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/ComplianceBanner/ComplianceBanner.stories.tsx]: added it in now

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx (hunk 2 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx]: added it in now

File: packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithMeetingId/JoinMeetingWithMeetingId.stories.tsx (hunk 3 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithMeetingId/JoinMeetingWithMeetingId.stories.tsx]: added it in now

File: packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx (hunk 4 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx]: turns out we need to add our "Stack" with the styles we are using our composite container stylings! 

added it in now

**Review by JoshuaLai — APPROVED** [review]

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**